hack/verify-golangci-lint.sh

```diff
@@ -21,15 +21,24 @@ set -o pipefail
 VERSION=v1.42.0
```

Comment:
and maybe we can bump the version to the latest as well

Reply:
Done!